Framework/PythonInterface/mantid/api/src/Exports/SpectrumInfoPythonIterator.cpp

```diff
@@ -7,7 +7,6 @@
 #include "MantidPythonInterface/api/SpectrumInfoPythonIterator.h"
 
 #include <boost/python/class.hpp>
-#include <boost/python/copy_const_reference.hpp>
 #include <boost/python/iterator.hpp>
 #include <boost/python/module.hpp>
```

Comment:
```suggestion
#include <boost/python/module.hpp>
#include <boost/python/reference_existing_object.hpp>
```
I know it compiles as it is, but seems safer to include the header. I can't actually see how that header comes into this file currently, must come from somewhere.